config/test.exs

```diff
@@ -16,7 +16,9 @@ config :changelog, Changelog.Mailer,
 # Configure your database
 config :changelog, Changelog.Repo,
   adapter: Ecto.Adapters.Postgres,
-  database: "changelog_test",
+  url: (
+    System.get_env("DB_URL") || "postgres://postgres@localhost:5432/changelog"
```

Comment:
Seems fine but why append `"_test"` to the end? Why not just have that be set in the environment variable?